components/FeatureMoblieView.tsx

```diff
@@ -19,7 +19,6 @@ export default function FeaturesMobileView() {
 
   const tabs = useRef<HTMLDivElement>(null);
   const codeRef = useRef<HTMLDivElement>(null);
```

Comment:
@maskedeman please check if this is still required?

Reply:
Thank you for pointing out.